src/services/report.ts

```diff
@@ -0,0 +1,83 @@
+import { devPrint } from "../components/utils/RandomUtils";
+import { RawReport, RawReportBody, Report, ReportBody } from "../types";
+import api from "./api";
+
+function serializedReport(report: RawReport): Report {
```

Comment:
try to use similar construct to other serializers if possible. also name should probably be deserializeReport since we're taking a "serialized" raw report and converting it to an application type